base_station/gui/src/components/AutonRoverMap.vue

```diff
@@ -225,7 +225,6 @@ export default {
     ...mapMutations('autonomy',{
       setClickPoint: 'setClickPoint',
       setWaypointList: 'setWaypointList',
-      setAutonMode: 'setAutonMode',
```

Comment:
This was not being used in the file, AutonRoverMap should not have access to this anyways